app/assets/javascripts/components/user_profiles/course_details.jsx

```diff
@@ -23,6 +23,10 @@ const CourseDetails = createReactClass({
               <div className="course-details_title">{I18n.t('courses.term')}</div>
               <div className="course-details_value">{course.course_term}</div>
             </div>
+            <div className="col">
+              <div className="course-details_title">{I18n.t('courses.academic_system')}</div>
+              <div className="course-details_value">{course.academic_system}</div>
```

Comment:
This data doesn't come from course.json.jbuilder, so academic system is not included.

I don't think this is especially useful info for the user profile page, though, so rather than add it to the json view this uses, we should just cut it.

Reply:
I'm adding this change and will merge this stuff shortly.